rdmo/management/management/commands/merge_attributes.py

```diff
@@ -0,0 +1,228 @@
+import logging
+from collections import namedtuple
+from textwrap import dedent
+
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.translation import gettext_lazy as _
+
+from rdmo.conditions.models import Condition
+from rdmo.domain.models import Attribute
+from rdmo.management.utils import replace_uri_in_template_string
+from rdmo.projects.models import Value
+from rdmo.questions.models import Page, Question, QuestionSet
+from rdmo.tasks.models import Task
+from rdmo.views.models import View
+
+logger = logging.getLogger(__name__)
+
+
+ModelHelper = namedtuple('ModelHelper', ['model', 'lookup_field', 'field_introspection', 'replacement_function'])
+
+REPLACE_ATTRIBUTE_MODEL_HELPERS = [
```

Comment:
This dict can be replaced with one:

```
if is_a_view:
    ...
else:
    ...
```

in then main loop.

Reply:
I've replaced these helpers with
```python
  related_model_fields = [i for i in Attribute._meta.get_fields() \
                          if i.is_relation and not i.many_to_many and i.related_model is not Attribute]
```
and handle the `View` separate and explicit.

seems agreeable to you?